lib/oli/activities/activity_registration.ex

```diff
@@ -18,6 +18,11 @@ defmodule Oli.Activities.ActivityRegistration do
     field :variables, {:array, :string}, default: []
     field :generates_report, :boolean, default: false
 
+    # Optionally, this activity registration can be associated with an LTI deployment.
+    # If an LTI deployment is associated, the activity is considered an LTI activity.
+    has_one :lti_external_tool_activity_deployment,
```

Comment:
Instead of adding an explicit field to identify the activity registration "subtype", an activity registration is simply identified as an LTI activity type by the presence of an `lti_external_tool_activity_deployment`